cfme/tests/configure/test_rest_access_control.py

```diff
@@ -66,7 +66,7 @@ def test_edit_tenants(self, appliance, tenants, multiple):
         assert tenants_len == len(edited)
         for index in range(tenants_len):
             record, _ = wait_for(
-                lambda: collection.find_by(name=new[index]['name']),
+                lambda: collection.find_by(name=new[index]['name']) or False,
```

Comment:
While this is one option, and I do like it, I at least wanted to make sure you were aware of the method of using the fail_condition argument (L101). Combined with the behavior of the fail condition check setting based on the type (L72) you might be able to get the behavior that you want without using the lambda at all or having to be a bit 'tricky'.

https://github.com/RedHatQE/wait_for/blob/master/wait_for/__init__.py#L101
https://github.com/RedHatQE/wait_for/blob/master/wait_for/__init__.py#L72

Once you've responded that you're aware of the argument/behavior ahead, but want to stick with this implementation I'm willing to merge. So totally optional but a bit more traditional to the method's intended use.